### PR TITLE
fix: start all the appropriate services for cloud tests in CI

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

Cloud tests are not connecting to kafka and so they do not run

In #10529 https://github.com/PostHog/posthog/pull/10529/files#diff-d534d457d2f453ff292b326309bebf3d6b2f7e067ca4247038b61df22d09c728R330 there was a typo leaving only one service starting rather than all services in the docker compose file

## Changes

fixes that

## How did you test this code?

by running the PR and seeing the tests pass

### CODE REVIEW

File: posthog/api/test/test_exports.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 
 class TestExports(APIBaseTest):
+    # add a comment to get python tests to run
```
Comment (pauldambra) [inline posthog/api/test/test_exports.py:29]:
```suggestion
```

File: posthog/api/test/test_exports.py [hunk 2]
```diff
@@ -26,6 +26,8 @@
...
+
```
Comment (pauldambra) [inline posthog/api/test/test_exports.py:30]:
```suggestion
```

**Review by fuziontech — APPROVED** [review]
image

**Comment by guidoiaquinti** [comment]
Thank you @pauldambra and sorry!

**Comment by pauldambra** [comment]
@guido non preoccuparti

Was fun to figure out 💖

**Pull request merged**